packages/components/src/form-toggle/test/index.js

```diff
@@ -17,16 +18,14 @@ describe( 'FormToggle', () => {
 				true
 			);
 			expect( formToggle.hasClass( 'is-checked' ) ).toBe( false );
-			expect( formToggle.type() ).toBe( 'span' );
+			expect( formToggle.type().__emotion_base ).toBe( 'span' );
```

Comment:
I really hate this change I had to make to allow this test to pass, reaching for a private, internal variable like this never feels good to me. Should this just be replaced by snapshot or is there something else we can test that gets at the same intention without depending on such a specific implementation detail?

Reply:
I agree with your sentiment. I think tests like these should either be snapshot or probably deleted 🙈